extensions/src/main/java/cm/aptoide/pt/extensions/DpExtensions.kt

```diff
@@ -0,0 +1,6 @@
+package cm.aptoide.pt.extensions
+
+import android.content.res.Resources
+import androidx.compose.ui.unit.Dp
+
+val Dp.toPx get() = this.value * Resources.getSystem().displayMetrics.density
```

Comment:
`toPx` is an action, and action should be a method, not value.
Alternatively rename it to a noun, like `pxSize` or `pxValue`